code/controllers/subsystem/sounds.dm

```diff
@@ -118,9 +118,7 @@ SUBSYSTEM_DEF(sounds)
 
 /// Random available channel, returns text.
 /datum/controller/subsystem/sounds/proc/random_available_channel_text()
-	if(channel_random_low > channel_reserve_high)
-		channel_random_low = 1
-	. = "[channel_list[channel_random_low++]]"
+	. = "[random_available_channel()]"
```

Comment:
Can you remove this for now so we can keep tg parity even if it is a code downgrade

Reply:
Done, don't tempt me to port this part to TG.